runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowElementExecutionTracker.java

```diff
@@ -311,6 +311,7 @@ private void attributeProcessingTime(Duration duration, long snapshot) {
             executionCount.getKey(),
             (unused, total) -> {
               int numExecutions = executionCount.getValue();
+              @SuppressWarnings("PreferJavaTimeOverload")
```

Comment:
Is this a false positive? I'm not sure what the check is expecting you to change here

Reply:
I think it was a false positive, I removed and it's not showing a warning now.